frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+{
+  "appName": "ऐप फ्लोवी",
+  "defaultUsername": "में",
+  "welcomeText": "स्वागत है @:appName",
+  "githubStarText": "गिटहब पैर स्टार दे",
+  "subscribeNewsletterText": "हमारे न्यूज़ की जानकारी पाए",
+  "letsGoButtonText": "शुरू करे",
+  "title": "टाइटल",
+  "signUp": {
+    "buttonText": "साईन करे",
+    "title": "साईन करे @:appName",
+    "getStartedText": "शुरू करे",
+    "emptyPasswordError": "पासवर्ड खाली नहीं रख सकते",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है",
+    "alreadyHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "repeatPasswordHint": "रिपीट पासवर्ड"
+  },
+  "signIn": {
+    "loginTitle": "लॉगिन करे @:appName",
+    "loginButtonText": "लॉगिन",
+    "loginAsGuestButtonText": "शुरू करे",
+    "buttonText": "साईन करे",
+    "forgotPassword": "क्या आप पासवर्ड भूल गए ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "dontHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है"
+  },
+  "workspace": {
+    "create": "काम करने की जगह बनाये",
+    "hint": "काम करने की जगह",
+    "notFoundError": "काम करने की जगह नहीं मिली"
```

Comment:
This is exactly why we need to replace `काम करने की जगह` with `वर्कस्पेस` because, here the current translation means that there is no space to work, while the actual meaning is that the app could not load a previously saved workspace.

Reply:
cc: @AmanNegi

Reply:
Yeah exactly in this case it's even more troublesome, notFoundError can be changed to something like यह वर्कस्पेस मौजूद नहीं है. Does this sound good @MayurSMahajan ?

Reply:
Yes!